src/Standards/Generic/Tests/ControlStructures/InlineControlStructureUnitTest.1.inc

```diff
@@ -88,17 +88,17 @@ if ($error === ERROR_INVALID_GENDER): ?>
 
 
 
-if ($this->allowShopping !== true):
-    if ($this->status != Shop_Cart :: OK):
-        switch ($this->status):
-            case Shop_Cart :: NOT_FOUND:
-            echo 'foo';
-        endswitch;
+if ($value):
+    if ($anotherValue):
+        foreach ($array as $element):
+            echo $element;
```

Comment:
That makes sense, I added the following: `echo !is_null($element) ? $element : 'default';`

Reply:
Ah, but the `:` in that statement would be tokenized as `T_INLINE_ELSE`, not as `T_COLON`. Want to have another look at this ?

Reply:
I misunderstood your suggestion during our call and should have read the comment more carefully.

I just pushed a new commit updating the code to use a `T_COLON` part of a function's return type declaration:

`echo (function($element): string { return $element; } )($element);`

The code is a bit weird in that it could be simplified to just `echo $element`, but that is probably ok for this test as it ensures there is a `T_COLON` token not related to an inline control structure alternative syntax.

Reply:
Convoluted code samples are a good thing™ in the context of tests for PHPCS, so 👍🏻